All project code/views/partials/calendar.ejs

```diff
@@ -82,6 +88,7 @@
             <input type="number" value=<%- addDate %> name="day" hidden>
             <input type="number" value=<%- addMonth %> name="month" hidden>
             <input type="number" value=<%- addYear %> name="year" hidden>
+            <input type="number" value=<%- (currWeek + 1) %> name="week" hidden>
```

Comment:
I think it might be clearer if we separate "currWeek" and "week".
I would have expected currWeek to be the week (1-52) in a year.

Maybe rename the variables currWeek and week to viewWeek or viewingWeek or similar?
(also perhaps remove the redundant variable representing the same thing, so using only one variable)

Reply:
Having a separate viewingWeek would make sense from currWeek if you instead make currWeek represent 1-52 that the current date is on the year, while viewWeek would be the week the user is currently viewing with your next/previous buttons.

The other variables though I'm failing to see any possible distinctions.

Reply:
I changed the name of the variable to be more clear, but a currWeek variable storing 1-52 is not necessary for the purpose of the calendar. The calendar doesn't serve a purpose to show what number week in the year they are viewing, the date is sufficient enough